indico/modules/events/sessions/models/blocks.py

```diff
@@ -127,9 +131,6 @@ def can_manage(self, user, allow_admin=True):
         else:
             return False
 
-    def can_access(self, user, allow_admin=True):
-        return self.session.can_access(user, allow_admin=allow_admin)
```

Comment:
Check this line ;)
https://github.com/indico/indico/pull/2296/files/22646562c0e83102506fe3cafe31f5bff9c031c6#diff-2852b3896b207ee61528fd8aba9593aeR102